lib/graphql/query.rb

```diff
@@ -254,6 +254,7 @@ def initialize(query, instrumenters)
       def call
         @instrumenters.each { |i| i.before_query(@query) }
         result = get_result
+      ensure
         @instrumenters.each { |i| i.after_query(@query) }
         result
```

Comment:
This line and the `result = ` assignment above isn't needed, since the result of the ensure block isn't returned.